frontend/src/views/CalendarPage/CalendarPage.jsx

```diff
@@ -0,0 +1,51 @@
+import './CalendarPage.scss';
+
+import { FormControl, MenuItem, Select } from '@mui/material';
+import { useState } from 'react';
+
+import MonthlyCalendar from '../../components/Calendar/MonthlyCalendar';
+import WeeklyCalendar from '../../components/Calendar/WeeklyCalendar';
+
+const CalendarPage = () => {
+  const [calendar, setCalendar] = useState(<MonthlyCalendar />);
+  const [calendarName, setCalendarName] = useState('Month');
+
+  const handleChange = (e) => {
+    setCalendarName(e.target.value);
+    switch (e.target.value) {
+      case 'Month':
+        setCalendar(<MonthlyCalendar />);
+        break;
+      case 'Week':
+        setCalendar(<WeeklyCalendar />);
+        break;
+      case 'Day':
+        setCalendar(<div>Daily Drugs</div>);
+        break;
+      default:
+        break;
+    }
```

Comment:
I wonder if keeping a component in a state is a good idea/not a bad practice. 
How about adding in the return code such as: 
```jsx
{calendarName === "Month" && <MonthlyCalendar />}
{calendarName === "Week" && <WeeklyCalendar/>}
etc.
```
I will make handleChange really short and readable